src/styles/blocks/calendar.scss

```
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  width: $calendar-size;
+  padding: $calendar-paddings;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @each $day, $margin in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $margin;
+    }
+  }
+
+  @for $i from 28 through $month-length {
+    &--month-length--#{$i} :nth-child(n + #{$i+1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    position: relative;
+    @include box-size($box_with);
+    background-color: #eee;
+    border: 1px solid black;
+    transition-duration: 0.5s;
```

Always list the styles that you apply transition for:
```suggestion
    transition-duration: background-color 0.5s, transform 0.5s;
```